x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/index_data_visualizer_view/index_data_visualizer_view.tsx

```diff
@@ -893,6 +898,7 @@ export const IndexDataVisualizerView: FC<IndexDataVisualizerViewProps> = (dataVi
                     updatePageState={setDataVisualizerListState}
                     getItemIdToExpandedRowMap={getItemIdToExpandedRowMap}
                     extendedColumns={extendedColumns}
+                    width={windowWidth - 300}
```

Comment:
:question: Could you elaborate how we end up with that exact calculation of window width - 300?

Reply:
Removed this usage and updated to use a size observer for the table instead here d0d2289.